script/prepare_artifacts.sh

```diff
@@ -1,11 +1,31 @@
 #!/usr/bin/env bash
 
-set -ex
+set -e
 
+# Create dir structure
+mkdir -p release .cr-index .cr-release-packages
+
+# Copy all deployment manifest in release directory
 for file in ./deployment/**/flyte_generated.yaml; do 
     if [ -f "$file" ]; then
         result=${file/#"./deployment/"}
         result=${result/%"/flyte_generated.yaml"}
         cp $file "./release/flyte_${result}_manifest.yaml"
     fi
 done
+
+# Download helm chart releaser
+wget -q -O /tmp/chart-releaser.tar.gz https://github.com/helm/chart-releaser/releases/download/v1.2.1/chart-releaser_1.2.1_linux_amd64.tar.gz 
+mkdir -p bin
+tar -xf /tmp/chart-releaser.tar.gz -C bin
+chmod +x bin/cr
+rm /tmp/chart-releaser.tar.gz \
+
+# Clean Git History
+git stash
+
+# Package helm chart
+bin/cr package helm
+
+# Commit Chart registry to github gh-pages
+bin/cr index --owner flyteorg --git-repo flyte --charts-repo="https://flyteorg.github.io/flyte" --push --token=${FLYTE_BOT_PAT} --release-name-template="{{ .Version }}"
```

Comment:
Where does it get the `Version` value from? I think they come from here: https://github.com/flyteorg/flyte/blob/master/helm/Chart.yaml#L5-L6 

Will these values be updated as part of this: https://github.com/flyteorg/flyte/actions/workflows/generate-flyte-manifest.yml ?

Reply:
yes, they come from the chart.yaml

Reply:
Yes, they are coming from chart.yaml and we can't add them in the above workflow because we need to add a version that we will create, and that workflow doesn't know that.


We need to update the value manually

Reply:
Maybe what we should do is have a placeholder `#VERSION` in the chart.yaml... and we populate that here, temporarily, using the version passed in to the current workflow. We package the chart and push it to the gh-pages branch then we `git stash` that change.... how about that?

Reply:
yes that will work

Reply:
done